agent/bench-scripts/tests/pbench-trafficgen/test-50.txt

```diff
@@ -74,7 +74,7 @@ Iteration 1-none-bidirectional-1500B-128flows-0.002pct_drop complete (1 of 1), w
 ./binary-search.py --traffic-generator=trex-txrx --rate-tolerance=3 --use-src-ip-flows=1 --use-dst-ip-flows=0 --use-src-mac-flows=0 --use-dst-mac-flows=0 --use-src-port-flows=0 --use-dst-port-flows=0 --use-encap-src-ip-flows=0 --use-encap-dst-ip-flows=0 --use-encap-src-mac-flows=0 --use-encap-dst-mac-flows=0 --use-protocol-flows=0 --device-pairs=0:1 --active-device-pairs=0:1 --sniff-runtime=30 --search-runtime=120 --validation-runtime=300 --max-loss-pct=0.002 --rate-unit=% --rate=100 --traffic-direction=bidirectional --frame-size=1500 --num-flows=128 --output-dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-50_tg:trex_r:none_fs:1500_nf:128_fm:si_td:bi_ml:0.002_tt:bs_1900.01.01T00.00.00/1-none-bidirectional-1500B-128flows-0.002pct_drop/sample2
 ./binary-search.py --traffic-generator=trex-txrx --rate-tolerance=3 --use-src-ip-flows=1 --use-dst-ip-flows=0 --use-src-mac-flows=0 --use-dst-mac-flows=0 --use-src-port-flows=0 --use-dst-port-flows=0 --use-encap-src-ip-flows=0 --use-encap-dst-ip-flows=0 --use-encap-src-mac-flows=0 --use-encap-dst-mac-flows=0 --use-protocol-flows=0 --device-pairs=0:1 --active-device-pairs=0:1 --sniff-runtime=30 --search-runtime=120 --validation-runtime=300 --max-loss-pct=0.002 --rate-unit=% --rate=100 --traffic-direction=bidirectional --frame-size=1500 --num-flows=128 --output-dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-50_tg:trex_r:none_fs:1500_nf:128_fm:si_td:bi_ml:0.002_tt:bs_1900.01.01T00.00.00/1-none-bidirectional-1500B-128flows-0.002pct_drop/sample3
-/var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/postprocess/generate-benchmark-summary trafficgen --config=test-50 --samples=3 --frame-sizes=1500 --num-flows=128 --traffic-directions=bidirectional --flow-mods=src-ip --traffic-generator=trex-txrx --devices=d8:00.0,d8:00.1 --search-runtime=120 --validation-runtime=300 --max-loss-pcts=0.002 --skip-git-pull --skip-trex-server /var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-50_tg:trex_r:none_fs:1500_nf:128_fm:si_td:bi_ml:0.002_tt:bs_1900.01.01T00.00.00
+/var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/postprocess/generate-benchmark-summary trafficgen --config=test-50 --samples=3 --frame-sizes=1500 --num-flows=128 --traffic-directions=bidirectional --flow-mods=src-ip --traffic-generator=trex-txrx --devices=d8:00.0,d8:00.1 --search-runtime=120 --validation-runtime=300 --max-loss-pcts=0.002 --skip-git-pull --skip-trex-server --config=test-50 /var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-50_tg:trex_r:none_fs:1500_nf:128_fm:si_td:bi_ml:0.002_tt:bs_1900.01.01T00.00.00
```

Comment:
The `--config=test-50` switch shows up twice in this list...which breaks the Jenkins test run.  And test-51 suffers a similar problem.

Reply:
hmm, must be a bad rebase ... thanks!

Reply:
Ugh, looks like I was in the middle of a `git rebase` operation and erroneously pushed before I had completed it.

Reply:
Should be fixed now.